master/internal/rm/kubernetesrm/informer.go

```diff
@@ -81,28 +54,27 @@ func (i *informer) startInformer(ctx *actor.Context) {
 		},
 	})
 	if err != nil {
-		ctx.Log().WithError(err).Warnf("error initializing pod retry watcher")
-		actors.NotifyAfter(ctx, defaultInformerBackoff, startInformer{})
-		return
+		i.syslog.WithError(err).Warnf("error initializing pod retry watcher")
+		ok = false
 	}
 
-	ctx.Log().Info("pod informer is starting")
+	i.syslog.Info("pod informer is starting")
 	for event := range rw.ResultChan() {
 		if event.Type == watch.Error {
-			ctx.Log().Warnf("pod informer emitted error %+v", event)
+			i.syslog.Warnf("pod informer emitted error %+v", event)
 			continue
 		}
 
-		pod, ok := event.Object.(*k8sV1.Pod)
-		if !ok {
-			ctx.Log().Warnf("error converting event of type %T to *k8sV1.Pod: %+v", event, event)
+		pod, OK := event.Object.(*k8sV1.Pod)
+		if !OK { // TODO CAROLINA: capitalizing OK to differentiate from ok var, better name?
```

Comment:
Maybe `ok` should be something more descriptive like `errorEncountered` since it spans the whole function.

Reply:
Personally I prefer `ok`, it is the idiomatic way for type assertions (ref https://go.dev/ref/spec#Type_assertions)

Reply:
I totally agree: `ok` is idiomatic for things like type assertions and the two-return-value get operation from `map`s. [Here](https://github.com/determined-ai/determined/blob/8cfd94a84d507b04795ae179c82680890449914b/master/internal/rm/kubernetesrm/informer.go#L42), `ok` was instead used to describe whether an error had been encountered, so it wasn't available to be used idiomatically for this type assertion.